src/cmd/build.rs

```diff
@@ -339,30 +375,32 @@ pub(crate) fn execute_with_crate_metadata(
     optimize_contract: bool,
     build_artifact: BuildArtifacts,
     unstable_flags: UnstableFlags,
-) -> Result<(Option<PathBuf>, Option<OptimizationResult>)> {
+    debug: bool,
+) -> Result<(Option<PathBuf>, Option<PathBuf>, Option<OptimizationResult>)> {
```

Comment:
This modification is quite dirty, would mind offering some advice if this PR is welcome in the next days? @Robbepop